FlowCrypt/Core/Core.swift

```diff
@@ -62,10 +56,17 @@ class Core: KeyDecrypter, KeyParser, CoreComposeMessageType {
 
     // MARK: - Setup
     func setupWebView() {
-        guard let jsFileSrc = getCoreJsFile() else { return }
-
-        let appVersion = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String ?? "[unknown version]"
-        webView.evaluateJavaScript("const APP_VERSION = 'iOS \(appVersion)';\(jsFileSrc)") { _, _ in }
+        DispatchQueue.main.async {
```

Comment:
@sosnovsky  I believe the app crashed because the webview was not set up on the main thread. When I tried to re-setup the webview in thread detail view, it threw an error stating that the webview setup code must be executed on the main thread.

Reply:
Waiting for my testflight build to throw error and new build to work fine.
Will let you know once I finish testing so that you can review

Reply:
Sounds good 👍
Maybe we can use `@MainActor` here?

Reply:
Sure. Maybe you need to build & run this PR in device and check after 8-10 hours if app crashes?